package.json

```diff
@@ -7,7 +7,8 @@
     "dev": "node server.js",
     "build": "next build",
     "start": "cross-env NODE_ENV=production node server.js",
-    "deploy": "now --public",
+    "deploy": "now -e NODE_ENV=production --public",
+    "deploy:ci": "npm run deploy -- --token $NOW_TOKEN --team dawn",
```

Comment:
Very cool! I want to set something up like this for some of my projects.
How did you assign `$NOW_TOKEN`? Is that somehow encrypted in the `.travis.yml` file?

Reply:
Hey @styfle It is encrypted under the `.travis.yml` under `secrets`. You can generate them using the [Travis CLI](https://github.com/travis-ci/travis.rb)